integration/src/test/scala/org/knora/webapi/e2e/v2/MarkupHeader.scala

```diff
@@ -9,25 +9,25 @@ import org.apache.pekko
 
 import scala.util.Try
 
-import org.knora.webapi.routing.RouteUtilV2
-
 import pekko.http.scaladsl.model.headers.ModeledCustomHeader
 import pekko.http.scaladsl.model.headers.ModeledCustomHeaderCompanion
 
 /**
- * A custom Pekko HTTP header representing [[RouteUtilV2.MARKUP_HEADER]], which a client can send to specify
+ * A custom Pekko HTTP header representing "x-knora-accept-markup", which a client can send to specify
  * how text markup should be returned in an API response.
  *
  * The definition follows [[https://doc.pekko.io/docs/pekko-http/current/common/http-model.html#custom-headers]].
  */
-final class MarkupHeader(token: String) extends ModeledCustomHeader[MarkupHeader] {
+final class MarkupHeader private (token: String) extends ModeledCustomHeader[MarkupHeader] {
   override def renderInRequests             = true
   override def renderInResponses            = true
   override val companion: MarkupHeader.type = MarkupHeader
   override def value: String                = token
 }
 
 object MarkupHeader extends ModeledCustomHeaderCompanion[MarkupHeader] {
-  override val name: String         = RouteUtilV2.MARKUP_HEADER
+  override val name: String         = "x-knora-accept-markup"
```

Comment:
Can somehow `org.knora.webapi.slice.common.api.ApiV2.Headers` be reused here and in below header files? You have also created in this PR.

Reply:
No, we cannot as this is test code here. The redundancy ensures that if the production code changes =>the test fails. Would we use the production constant here and it changes the test would still succeed.